plugins/BEdita/API/src/Auth/EndpointAuthorize.php

```diff
@@ -96,6 +97,20 @@ public function authorize($user, ServerRequest $request)
             $this->unauthenticate();
         }
 
+        // if 'administratorOnly' configuration is true logged user must have administrator role
```

Comment:
I suggest this simpler change just before  `if ($this->isAnonymous($user) && $this->authorized !== true)`

```php
if ($this->authorized && $this->getConfig('administratorOnly')) {
 $this->authorized = in_array(self::ADMIN_ROLE, Hash::extract($user, 'roles.{n}.id'));
} 
```

a definition of `const ADMIN_ROLE = 1;` should be added to class

Explanation: 
 * if user is unlogged `$this->authorized` will be false and user will get a 401 anyway
 * if user is logged but without administrator role response will be 403